extensions/arc/src/ui/dialogs/upgradeSqlMiaa.ts

```diff
@@ -0,0 +1,93 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import { Deferred } from '../../common/promise';
+import * as loc from '../../localizedConstants';
+import * as vscode from 'vscode';
+import { cssStyles } from '../../constants';
+import { InitializingComponent } from '../components/initializingComponent';
+import { UpgradeModel } from '../../models/miaaModel';
+import { ControllerModel } from '../../models/controllerModel';
+
+export class UpgradeSqlMiaa extends InitializingComponent {
+	protected modelBuilder!: azdata.ModelBuilder;
+	private pitrSettings: UpgradeModel = [];
+
+	protected _completionPromise = new Deferred<UpgradeModel | undefined>();
+	protected disposables: vscode.Disposable[] = [];
+	constructor(protected _controllerModel: ControllerModel) {
+		super();
+	}
+
+	public showDialog(dialogTitle: string): azdata.window.Dialog {
+		const dialog = azdata.window.createModelViewDialog(dialogTitle, dialogTitle, 'narrow', 'flyout');
```

Comment:
The name shouldn't be the same as the title - it should be a constant unlocalized value such as `UpgradeSqlMiaaDialog`